.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore hte folder node_modules
+node_modules
+# Ignore the folder test.md
+.md
```

Comment:
- [ ] According to the project requirement, you are also meant to add the `test.md` to the .gitignore file, I'll suggest you follow the requirement and do that. :+1: